content/rs/release-notes/rs-7-4-2-releases/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+Title: Redis Enterprise Software release notes 7.4.2
+linkTitle: 7.4.2 releases
+description: New Cluster Manager UI enhancements, including Active-Active database management. TLS 1.3 support. Automatic recovery configuration. IPv6 support for internal traffic. Maintenance mode enhancements. Two feature sets.
+compatibleOSSVersion: Redis 7.2.0
+weight: 70
+alwaysopen: false
+toc: "true"
+categories: ["RS"]
+---
+
+​[​Redis Enterprise Software version 7.4.2](https://redis.com/redis-enterprise-software/download-center/software/) is now available!
+
+## Highlights
+
+This version offers:
+ 
+- New Cluster Manager UI enhancements, including Active-Active database management
+
+- TLS 1.3 support
+
+- Automatic recovery configuration
+
+- IPv6 support for internal traffic
+
+- Maintenance mode enhancements
```

Comment:
@rrelledge I'm not sure it's just enhancement, there is some changed behavior:
1. not storing multiple MM snapshots
2. deprecating the keep_slave_shards and moving to use with the new flags
3. snapshot added to node object
i'm just not sure of the wording

Reply:
@maayanagranat I added these details to https://docs.redis.com/staging/DOC-2982/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba/#enhancements. Does that help, or do you think I still need to reword the highlight a bit?